redbeams/src/main/java/com/sequenceiq/redbeams/service/RedbeamsConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.sequenceiq.redbeams.service;
+
+public class RedbeamsConstants {
```

Comment:
I'm not totally convinced this class should exist -- this constant string could go into DatabaseConfigService pretty easily. I don't feel strongly about that though, so I'll leave it to your judgment to decide what to do. Regardless, I'd like a Javadoc header on the constants if it remains.

Reply:
Ah yes. I wanted to refactor the "success" response from both classes into a common constant. I just forgot to add that to databaseserver.

Reply:
I then leave it as it is.